src/shared/locales/all.js

```diff
@@ -16,6 +18,12 @@ const resources = {
       ...appLocaleEnUS
     }
   },
+  'pr-BT': {
```

Comment:
Please make sure you can switch languages successfully.
I find a typo: (
`pr-BT` => `pt-BR`

Reply:
Sorry. Fixed!